benchmarks/TFLite/linux-riscv.cmake

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+################################################################################
+#                                                                              #
+# Default benchmark configurations                                             #
+#                                                                              #
+# Each suite benchmarks a list of modules with configurations specifying a     #
+# target architecture and runtime characteristics (e.g. threads/cores). These  #
+# benchmarks only configure IREE translation and runtime flags for the target  #
+# architecture and do *not* include any non-default flags. No non-default      #
+# flags should be added here.                                                  #
+#                                                                              #
+################################################################################
+
+set(LINUX_RISCV64_GENERIC_RV64_CPU_TRANSLATION_FLAGS
+  "--iree-input-type=tosa"
+  "--iree-llvm-target-triple=riscv64"
+  "--iree-llvm-target-cpu=generic-rv64"
+  "--iree-llvm-target-abi=lp64d"
+  "--iree-llvm-target-cpu-features=+m,+a,+f,+d,+c"
+)
+
+# CPU, Dylib-Sync, RISCV64, full-inference
```

Comment:
Currently the benchmark framework doesn't support only compiling modules, but there is no harm to have runtime configs here, since we don't run them in the pipeline. The rules I add here are just for compiling the RV64 modules.

I'm adding the script to collect compilation time and binary size and it will work with the module generated from here (#9227).

Reply:
I see from #9227 we only get the module size. However, I wonder if we should report both the bytecode module size as well as the embedded workload library size.

Reply:
I think it is a good idea to also track the embedded library size. The issue is I don't think our current infra can handle that many metrics well. So if the module size is good enough for now, I'd prefer to postpone the embedded library size.

I'm working on a new benchmark infra design and we should be able to support much more metrics with it. Happy to chat offline if you are interested in it : )

Reply:
Sorry, I saw this after submitting https://github.com/google/iree/pull/9218#discussion_r883123007. Yes, please, let's focus on just one metric for now, either the `.so` size of the bytecode size would work. Whatever is simpler.